README.md

```diff
@@ -1,10 +1,25 @@
 # TitanicML
-### Titanic predictions with Machine Learning comparing Logistic Regression, Random Forests and Gradient Boosted Trees from TensorFlow. Including data treatment and exploratory data analysis.
+
+Titanic predictions with Machine Learning comparing Logistic Regression, Random Forests and Gradient Boosted Trees from TensorFlow. Including data treatment and exploratory data analysis.
+
 Exploring the Kaggle Titanic Dataset to predict passengers survival.
 
 ##
 
-### 1. The notebook starts with Exploratory Data Analysis (EDA) where we can visualize the data, found possible errors in it, missing values, look for hints for the better features, and more. 
+### Summary
+
+1. [EDA](###-1.-the-notebook-starts-with-exploratory-data-analysis-(eda))
+2. Data Preprocessing
+3. Logistic Regression
+4. Random Forest
+5. Gradient Boosted Trees
+6. Results and Conclusions
+
+##
```

Comment:
To create a line in markdown you can type `---`

Reply:
Nice, like it's the same to use ## or --- !?

Reply:
Usually the character # its used only for headers you want to create.